### ISSUE TITLE
SaveObject() test fails under windows

### ISSUE THREAD

**Issue by fvaneijk** [fredericaltorres/Jint.Ex#3]
Assert.AreEqual failed. Expected:<{"a":1,"b":true,"c":"hello","d":"1990-01-01T06:02:03.000Z"}>. Actual:<{}>. 
   at JintEx_UnitTests.LocalStorageUnitTests.SaveObject() in LocalStorageUnitTests.cs: line 188

Is it possible its an issue with getItem() in LocalStorage.cs?

**Comment by fredericaltorres** [comment]
I ran the test yesterday, I will try again tonight

**Comment by fredericaltorres** [comment]
Yes there was an issue. I made a change in Jint to build for iOS and my change is not yet part of Jint.
This require from my part a manual step to build for iOS or Windows.
This should work now.

**Comment by fvaneijk** [comment]
Thanks for checking into it.

**Comment by fvaneijk** [comment]
Just confirmed it passes the tests now.  Thanks.
